app/Http/Controllers/Administration/SettingsController.php

```diff
@@ -300,6 +300,23 @@ public function setLocationShowPublic(Request $request)
 		) ? 'true' : 'false';
 	}
 
+	/**
+	 * Enable sending of new photos notification emails.
+	 *
+	 * @param Request $request
+	 *
+	 * @return string
+	 */
+	public function setNewPhotosNotification(Request $request)
+	{
+		$request->validate(['new_photos_notification' => 'required|string']);
+
+		return Configs::set(
+			'new_photos_notification',
+			$request['new_photos_notification']
```

Comment:
Ideally you want to check that the `new_photo_notification` is either `1` or `0`.